new-lamassu-admin/src/pages/Notifications/sections/CryptoBalanceOverrides.js

```diff
@@ -56,12 +56,10 @@ const CryptoBalanceOverrides = ({ section }) => {
     [LOW_BALANCE_KEY]: Yup.number()
       .integer()
       .min(0)
-      .max(99999999)
```

Comment:
This is the same that we talked about on other review and will be removed, right?

Reply:
Yup, should be removed already: https://github.com/lamassu/lamassu-server/pull/395/commits/a5182d76266de51cf6c3983d4dd90022b2a6b29b